setup-tests.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+import os
+
+os.system('pip install pipenv')
```

Comment:
I prefer if we just provide a complete requirements.txt file, which the user can pip install by:
`pip install -r requirements.txt`

Reply:
There are two advantages to the Pipenv approach:
- It does its installations in a virtualenv, so the main Python env is unaffected.
- It records current state of package version in the Pipenv.lock file, so a deterministic build state can be restored (this may not be very important in our case).
So, we need to decide whether to go with new Pipenv or stick with requirements.txt. I would, however, join all requirements.txt files into a single one.